synapse/lib/stormlib/json.py

```diff
@@ -116,24 +117,25 @@ def getObjLocals(self):
         }
 
     @s_stormtypes.stormfunc(readonly=True)
-    async def _jsonSave(self, item, indent=None):
-        indent = await s_stormtypes.toint(indent, noneok=True)
+    async def _jsonSave(self, item, indent=False):
+        try:
+            # A little bit of hackery for backwards compatibility with the old indent parameter being an integer/None
+            indent = bool(await s_stormtypes.toint(indent, noneok=True))
+        except s_exc.BadCast:
+            indent = await s_stormtypes.tobool(indent)
 
         try:
             item = await s_stormtypes.toprim(item)
-            return json.dumps(item, indent=indent)
-        except Exception as e:
+        except Exception:
             mesg = f'Argument is not JSON compatible: {item}'
             raise s_exc.MustBeJsonSafe(mesg=mesg)
 
+        return s_json.dumps(item, indent=indent)
```

Comment:
We're still going to have to handle integer values for indentation; which means we may need to do that ourselves.

Reply:
Integer values are handled above in the `bool(toint(...))`.

Reply:
They're converted to the orjson default behavior if provided but it breaks the storm API output if a user expected a different value for indentation.

Reply:
so fallback to the stdlib json if something other than (True, 2) is provided?